client/message/__tests__/view-test.js

```diff
@@ -6,18 +6,58 @@ jest.dontMock('../view.jsx');
 const Message = require('../view.jsx');
 
 describe('Message', function() {
-    it('renders a message', function() {
-        const message = TestUtils.renderIntoDocument(
-            <Message username='hilbert'
-                     own='false'
-                     text='Wir müssen wissen — wir werden wissen!' />
-        );
+    function renderAndGetNode(xml, node) {
+        const reactNode = TestUtils.renderIntoDocument(xml);
+        const domNode = TestUtils.findRenderedDOMComponentWithTag(
+            reactNode,
+            node
+        ).getDOMNode();
 
-        const li = TestUtils.findRenderedDOMComponentWithTag(
-            message,
+        return domNode;
+    }
+
+    function renderMessage(username, own, text, typing) {
+        return renderAndGetNode(
+            <Message username={username}
+                     own={own}
+                     text={text}
+                     typing={typing} />,
             'li'
-        ).getDOMNode();
+        );
+    }
+
+    it('renders a message', function() {
```

Comment:
No. It should be this way. Take a look at the style in which jasmine tests are typically written. For example:

http://evanhahn.com/how-do-i-jasmine/

This is borrowed from rspec, take a look:

http://code.tutsplus.com/tutorials/ruby-for-newbies-testing-with-rspec--net-21297